spec/index.bs

```diff
@@ -4428,26 +4435,52 @@ typedef double GPUPipelineConstantValue; // May represent WGSL's bool, f32, i32,
                 : {{GPUBindGroupLayoutEntry/sampler}}
                 :: If |entry|.{{GPUBindGroupLayoutEntry/sampler}}.{{GPUSamplerBindingLayout/type}} is:
                     <dl class=switch>
-                        : {{GPUSamplerBindingType/"filtering"}}
-                        :: the |binding| is a non-comparison sampler
-                        : {{GPUSamplerBindingType/"non-filtering"}}
-                        :: the |binding| is a non-comparison sampler
+                        : {{GPUSamplerBindingType/"filtering"}} or {{GPUSamplerBindingType/"non-filtering"}}
+                        :: |binding| has type `sampler`.
                         : {{GPUSamplerBindingType/"comparison"}}
-                        :: the |binding| is a comparison sampler
+                        :: |binding| has type `comparison_sampler`.
                     </dl>
 
                 : {{GPUBindGroupLayoutEntry/texture}}
-                :: If |entry|.{{GPUBindGroupLayoutEntry/texture}}.{{GPUTextureBindingLayout/multisampled}} is:
+                :: If, and only if,
+                    |entry|.{{GPUBindGroupLayoutEntry/texture}}.{{GPUTextureBindingLayout/multisampled}}
+                    is `true`, |binding| has type `texture_multisampled_2d<T>`.
+                :: If |entry|.{{GPUBindGroupLayoutEntry/texture}}.{{GPUTextureBindingLayout/sampleType}} is:
+                    <dl class=switch>
+                        : {{GPUTextureSampleType/"float"}}, {{GPUTextureSampleType/"unfilterable-float"}},
+                            {{GPUTextureSampleType/"sint"}} or {{GPUTextureSampleType/"uint"}}
+                        :: |binding| has type `texture_1d<T>`, `texture_2d<T>`, `texture_2d_array<T>`,
+                            `texture_cube<T>`, `texture_cube_array<T>`, `texture_3d<T>`, or
+                            `texture_multisampled_2d<T>`.
+                        :: If |entry|.{{GPUBindGroupLayoutEntry/texture}}.{{GPUTextureBindingLayout/sampleType}} is:
```

Comment:
This was really confusing to parse in the bikeshed doc, but makes a lot more sense when looking at the rendered spec.  I think a lot of my confusion was around the difference between ":" and "::" wasn't particularly easy to visually distinguish.

I don't have any suggestions on how to make it better, mostly pointing out that this was the only thing I tripped on while reviewing.

Reply:
Added some whitespace to hopefully make this clearer

```
: case
::
    item
::
    item
```